Changed rvm installation method

KarenAvetyan opened the pull request:
Changed rvm installation method according to official documentation.
https://github.com/rvm/ubuntu_rvm

KarenAvetyan committed:
Changed rvm installation method

Changed rvm installation method according to official documentation.
https://github.com/rvm/ubuntu_rvm

KarenAvetyan committed:
Update prepare_buildhost.sh

jhyoty commented:
There are some problems:
* On a clean build host, the prepare_buildhost script fails since ~/.rvm/scripts/rvm does not exist (there are also other scripts that assume that rvm is installed locally).
* The preparation scripts now requires interaction (breaks unattended build setup)

iluwatar reviewed (changes requested):
@KarenAvetyan As such the pull request can't be integrated. The X-Road main repository is being transferred to https://github.com/nordic-institute, so please make a new pull request there.

KarenAvetyan committed:
Update prepare_buildhost.sh

I've forgot that after installing rvm users need to relogin.
Now it works on clean system.
I will also change other scripts to run systemwide installed rvm package and commit it to new repository.
